content/foundations/color/base-scales.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Base scales
+description:
+  Primer base color scales makeup the foundation for all UI color considerations.
+---
+
+## All color scales
```

Comment:
Curious if we should leave an extra warning here. 

**Note:** Don't use these values directly. These values should only be used when constructing `Functional` and `Component/pattern` values.